### PR TITLE
OS-9

### PR DESCRIPTION
**Pull request by YevheniiH**

### CODE REVIEW

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        }
+
+        .pagebuilder-column-group.footer-wrapper {
+            .lib-vendor-prefix-display();
+            padding: 45px 20px;
+
+            .pagebuilder-column-line {
+                &:first-child {
+                    width: 50% !important;
+                }
```
Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less]: There is no explanation for a !important rule and I guess that if it's not inline style to rewrite, you may use #html-body id for overriding page-builder styles

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+                &:nth-child(2),
+                &:last-child {
+                    width: 25% !important;
```
Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less]: Same as above

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+        .footer.content {
+            display: none;
+        }
+
+        .pagebuilder-column-group.footer-wrapper {
+            .area-first,
+            .area-second,
+            .area-third {
+                h4 {
```
Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:18]: Why don't you use one common class name for all these 3 blocks?

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+                &:nth-child(2),
+                &:last-child {
+                    width: 25% !important;
+                }
+            }
+
+            .area-first,
+            .area-second,
+            .area-third {
+                margin: 0 10px;
```
Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:156]: Same thing when you see that this construction is taking place, think about adding one common class

File: src/app/design/frontend/Hanzha/OrganicStore/Magento_Theme/templates/html/copyright.phtml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+ */
+
+/** @var \Magento\Framework\View\Element\Template $block */
+$blockId = 'copyright_block';
+$cmsBlock = $block->getLayout()->createBlock(\Magento\Cms\Block\Block::class)->setBlockId($blockId);
+?>
+<small class="copyright">
+    <span><?= $block->escapeHtml($block->getCopyright()) ?></span>
+    <?php echo $cmsBlock->toHtml(); ?>
+</small>
```
Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Magento_Theme/templates/html/copyright.phtml:14]: Is this template default? I can't see the changes that you've made here. Why then you have a copyright cms block name in the scrtipt?

**Review by AndriiBatoh — CHANGES_REQUESTED** [review]

File: src/app/design/frontend/Hanzha/OrganicStore/Magento_Theme/templates/html/copyright.phtml (hunk 5 above)
Comment (YevheniiH) [reply src/app/design/frontend/Hanzha/OrganicStore/Magento_Theme/templates/html/copyright.phtml:14]: Thanks! Changed the template, to reflect the changes.

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less (hunk 1 above)
Comment (YevheniiH) [reply src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less]: Done. I used one generic class name

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less (hunk 4 above)
Comment (YevheniiH) [reply src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:156]: Changed!

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less (hunk 3 above)
Comment (YevheniiH) [reply src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:18]: OK! Changed! used one common class name.

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less (hunk 2 above)
Comment (YevheniiH) [reply src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less]: Done

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less [hunk 6]
```diff
@@ -0,0 +1,201 @@
+// /**
+//* Copyright © Batoh Andrii, All rights reserved.
+//* See COPYING.txt for license details.
+//* @author Yevhenii Hanzha <evgenijganza369@gmail.com>
+//  */
+& when (@media-common = true) {
+    .page-footer {
+        background: @color-os-black1 !important;
+
```
Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:9]: Review this rule

Comment (AndriiBatoh) [inline src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:10]: Review this rule

**Review by AndriiBatoh — CHANGES_REQUESTED** [review]

File: src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less (hunk 6 above)
Comment (YevheniiH) [reply src/app/design/frontend/Hanzha/OrganicStore/Hanzha_Cms/web/css/source/cms_block/_footer.less:10]: Changed!

**Review by AndriiBatoh — APPROVED** [review]

**Review by AndriiBatoh — APPROVED** [review]

**Pull request closed without merging**